docs/usage/operations.rst

```diff
@@ -6,7 +6,7 @@ Provides custom migration operations that help developers perform idempotent and
 Class Definitions
 -----------------
 
-.. py:class:: SaferAddIndexConcurrently(model_name: str, index: models.Index)
+.. py:class:: SaferAddIndexConcurrently(model_name: str, index: models.Index, hints: Any = None)
```

Comment:
❓  Should the type for `hints` be `Any` or `object`?

Reply:
I should've left a comment on the Pull Request for clarity, but essentially I am keeping the same type annotation as `django-stubs` for [consistency](https://github.com/typeddjango/django-stubs/blob/2dfa15bf94b038c4dae1590093981d3eacfa83e6/django-stubs/db/utils.pyi#L51).